dashboards/system_tables/sql/job_analyzer_slow.sql

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
I believe this file is the same as the "fast" one. Is there a way we can maintain a single copy of the source?

Reply:
done.